src/bp-forums/common/widgets.php

```diff
@@ -591,7 +591,7 @@ public function widget( $args, $instance ) {
 			array(
 				'post_type'           => bbp_get_forum_post_type(),
 				// 'post_parent'         => $settings['parent_forum'],
```

Comment:
@kent-carlo This is the same line that you added so can you check history why it's removed.

Reply:
@dipskakadiya It only says "forums list widgets updates", should I remove it? This is the commit https://github.com/buddyboss/buddyboss-platform/commit/8ac8a622044ad82a20f95293935d9fe194df90bd